install-dev.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 # called from Pipfile
-pipenv run pip install https://github.com/openwisp/django-netjsonconfig/tarball/master
-pipenv run pip install --upgrade "openwisp-utils[qa]>=0.3.2,<0.4.0"
+pipenv run pip install --upgrade https://github.com/openwisp/django-netjsonconfig/tarball/master
+pipenv run pip install --upgrade https://github.com/openwisp/django-loci/tarball/master
```

Comment:
I just released [django-loci 0.3.1](https://pypi.org/project/django-loci/0.3.1/) so you can now remove this line and set the minimum django-loci version to 0.3.1